debian/nfd.postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+if ! ndnsec-get-default >/dev/null
+then # generate keys for this machine
+  ndnsec-keygen /localhost/operator | ndnsec-install-cert -
```

Comment:
we should **also** ship the endless keys.
this makes sense for debian, for our case we're going to need to fix it somewhere else =/...